src/components/appheader.main.jsx

```diff
@@ -185,7 +208,7 @@ class AppHeaderMain extends React.Component {
               <span className="icon glyphicon glyphicon-align-justify" />
             </button>
           </div>
-
+          <BulkOrderMain isBulkModalOpened={isBulkModalOpened} handleClose={this.handleBulkModalClose} cartData={cartData} />
```

Comment:
We should only show this if b2b is enabled. Even though the API is available on b2c as well, we want to only demonstrate it in b2b.